lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -771,5 +810,36 @@ export abstract class ClientApplication {
         return validLogoutRequest;
     }
 
+    /**
+     * Broadcasts events by calling callback with broadcast message
+     * @param eventType 
+     * @param interactionType 
+     * @param payload 
+     * @param error 
+     */
+    broadcastEvent(eventType: BroadcastEvent, interactionType?: InteractionType, payload?: BroadcastPayload, error?: BroadcastError) {
+        const message: BroadcastMessage = {
+            eventType: eventType,
+            interactionType: interactionType || null,
+            payload: payload || null,
+            error: error || null,
+            timestamp: Date.now()
+        };
+
+        this.eventCallbacks.forEach((callback) => {
+            callback(message);
```

Comment:
I think a verbose log message would be useful here (a message indicating an event with the given eventType is probably good enough to start).

Reply:
Done.